assets/css/_skate_ui.scss

```diff
@@ -108,6 +109,70 @@ $font-weights: (
   line-height: 1;
 }
 
+/* Input Submit Button */
+
+.button-submit {
+  color: $white;
+  background-color: $color-eggplant-700;
+
+  &:hover {
+    background-color: $color-eggplant-500;
+  }
+
+  &:focus {
+    background-color: $color-eggplant-500;
+    border: 2px solid $color-eggplant-700;
+  }
+
+  &:disabled {
+    background-color: $color-gray-600;
+  }
+}
+
+/* Search Filters */
```

Comment:
I moved it here because these are now actually based on a standard component (see "Search Filters" in [this Figma file](https://www.figma.com/file/zhyuIYsYLnftCswTg5AM8i/Skate-%7C-Component-Library?node-id=9764%3A35644&t=qNVdN6CWaeLsqoWG-0)).

As for `_skate_ui` vs. `_ui_kit` I'm not sure, the former was just added in #1873. @firestack since you initially added it do you have any opinions on which sorts of things should go in which file?

Reply:
I originally had the changes from `_skate_ui` in `_ui_kit` but moved them out because it was difficult to organize them in a way that made sense, and ui_kit seemed at the time of reading to have a lot of ideas that may not be current design standards anymore. My idea of creating `_skate_ui` was to have a set of standard typography and colors to share with design to ease the work of translating designs into css styles. 

If we have a place where design says that we'll be using the same style in lots of places, I think it makes sense to have it in `skate_ui`?

Reply:
I do like separating out new standard UI components from the old ones in _ui_kit.scss`, a lot of those mixins and stuff haven't been used to implement a new component in a long time.